src/azure-cli/requirements.py3.windows.txt

```diff
@@ -120,7 +120,7 @@ pycomposefile==0.0.30
 pycparser==2.19
 PyGithub==1.55
 PyJWT==2.4.0
-pymsalruntime==0.14.0
+pymsalruntime==0.15.0a1
```

Comment:
MSAL 1.28.0 requires:

https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/1.28.0/setup.cfg#L65

```py
    pymsalruntime>=0.13.2,<0.15; python_version>='3.6' and platform_system=='Windows'
```

So `pymsalruntime==0.15.0a1` is not allowed. 

`pip` fails:

```
ERROR: Cannot install msal[broker]==1.28.0 and pymsalruntime==0.15.0a1 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested pymsalruntime==0.15.0a1
    msal[broker] 1.28.0 depends on pymsalruntime<0.15 and >=0.13.2; (python_version >= "3.6" and platform_system == "Windows") and extra == "broker"
```

Reply:
PyMSALRuntime's version has been reverted to 0.14.2a1: https://pypi.org/project/pymsalruntime/0.14.2a1/